nbs/A_B.ipynb

```diff
@@ -270,6 +261,7 @@
         "# not a fan of this name. maybe call this section \"experiments\"?\n",
         "differentiators:\n",
         "  test_case_A:\n",
+        "  host: grpc-staging.stability.ai:443\n",
```

Comment:
pretty sure we don't want this endpoint to be the notebook default, since this notebook will be public and I'm pretty sure this endpoint isn't supposed to be. for cases where we want to test against this endpoint, we can distribute an appropriate config file that has it as a test case like this

Reply:
Oh I see. Very thoughtful. Thanks!